app/src/main/java/com/framgia/forder/screen/listProduct/adapter/ListProductAdapter.java

```diff
@@ -3,96 +3,103 @@
-            default:
-                break;
-        }
-        return null;
+    public ListProductAdapter.ItemViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        ItemListProductBinding binding =
+                DataBindingUtil.inflate(LayoutInflater.from(parent.getContext()),
+                        R.layout.item_list_product, parent, false);
+        return new ListProductAdapter.ItemViewHolder(binding, mItemClickListener);
     }
 
     @Override
-    public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
-        switch (getItemViewType(position)) {
-            case HEADER:
-                HeaderViewHolder headerViewHolder = (HeaderViewHolder) holder;
-                headerViewHolder.bind(mProducts.get(position));
-                break;
-            case CONTENT:
-                ContentViewHolder contentViewHolder = (ContentViewHolder) holder;
-                contentViewHolder.bind(mProducts.get(position));
-                break;
-        }
+    public void onBindViewHolder(final ListProductAdapter.ItemViewHolder holder, int position) {
+        holder.bind(mProducts.get(position));
+        holder.mImageMoreOrder.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showPopupMenu(holder.mImageMoreOrder);
+            }
+        });
+        mAnimationManager.animationSlideInLeft(holder.itemView, position);
     }
 
     @Override
     public int getItemCount() {
         return mProducts.size();
     }
 
-    public void setOrderListener(OrderListener listener) {
-        mOrderListener = listener;
-    }
-
     public void setItemClickListener(OnRecyclerViewItemClickListener<Object> itemClickListener) {
         mItemClickListener = itemClickListener;
     }
 
-    @Override
-    public int getItemViewType(int position) {
-        if (position == 0) {
-            return HEADER;
-        } else {
-            return CONTENT;
+    private void showPopupMenu(View view) {
+        PopupMenu popupMenu = new PopupMenu(getContext(), view);
+        MenuInflater menuInflater = popupMenu.getMenuInflater();
+        menuInflater.inflate(R.menu.menu_order, popupMenu.getMenu());
+        popupMenu.setOnMenuItemClickListener(new MyMenuItemClickListener());
+        popupMenu.show();
+    }
+
+    private class MyMenuItemClickListener implements PopupMenu.OnMenuItemClickListener {
+
+        @Override
+        public boolean onMenuItemClick(MenuItem item) {
+            switch (item.getItemId()) {
+                case R.id.add_to_cart:
+                    // TODO Click Add To Cart
+                    return true;
+                case R.id.order_now:
+                    // TODO Click Order Now
+                    return true;
+                default:
+            }
+            return false;
         }
     }
 
+    public void setOrderListener(OrderListener orderListener) {
+        mOrderListener = orderListener;
```

Comment:
ERROR: [STYLE][ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD] Write to static field com.framgia.forder.screen.listProduct.adapter.ListProductAdapter.mOrderListener from instance method com.framgia.forder.screen.listProduct.adapter.ListProductAdapter.setOrderListener(OrderListener)
http://findbugs.sourceforge.net/bugDescriptions.html#ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD